ts/src/renderer/phaser/classes/PhaserAttributeBar.ts

```diff
@@ -103,7 +104,7 @@ class PhaserAttributeBar extends Phaser.GameObjects.Container {
 		const sprite = this.unit.sprite;
 		this.y = 25 +
 			Math.max(sprite.displayHeight, sprite.displayWidth) / 2
-			+ data.index * h*1.1;
+			+ (data.index - 1) * h*1.1;
```

Comment:
index inside PhaserAttributeBar is for all cases, change we made inside unit,js - just when updating values, so for now it will be same index for all cases. this change inside PhaserAttributeBar  is just cosmetics btw, attribute bar will render a bit higher